docs/background/case-studies/heritage-impact-assessment.md

```diff
@@ -0,0 +1,489 @@
+---
+title: Heritage Impact Assessment
+---
+
+# Heritage Impact Assessment using QField
+
+## From Qgis to Qfield and Vice Versa: How the New Android Application Is Facilitating the Work of the Archaeologist in the Field
+
+**Roberto Montagnetti 1,* and Giuseppe Guarino 2**
+
+<sub><sup>1 adArte s.r.l. Archeologia, Restauro, ICT, 47921 Rimini, Italy</sup></sub>
+
+<sub><sup>2 Department of History and Cultures—DISCI, University of Bologna, 40124 Bologna, Italy;
+guarino.archeo@gmail.com</sup></sub>
+
+<sub><sup>* Correspondence: robertomontagnetti@gmail.com</sup></sub>
+
+<sub><sup>† Presented at the ArcheoFOSS XIII Workshop—Open Software, Hardware, Processes, Data and Formats in
+Archaeological Research, Padova, Italy, 20–22 February 2019.</sup></sub>
+
+<sub><sup>**Abstract:** The aim of this paper is to highlight the main benefits of using the Qfield app in archae-
+ological jobs. In particular this article provides examples to use Qfield in open area excavation,
+Archaeological survey and impact assessment (HIA) projects.</sup></sub>
+
+<sub><sup>**Keywords:** Qfield; archeology; VIARCH; HIA; Qgis</sup></sub>
+
+----------------------------------------------------------------------------------------------------------------------
+
+### 1. Introduction
+
+The aim of this paper is to highlight the main benefits of using the Qfield app. An

```

Comment:
Can we have a consistent spelling of "QField" and "QGIS" (capitalisation)?

Reply:
I took the original text, I didn't want to change it. But if you think it's fine to do it, I will.

Reply:
I think it's ok to adjust it